src/components/tree-item/tree-item.e2e.ts

```diff
@@ -292,4 +292,59 @@ describe("calcite-tree-item", () => {
     expect(checkbox).not.toHaveAttribute("checked");
     expect(isVisible).toBe(true);
   });
+
+  it("right arrow key expands subtree and left arrow collapses it", async () => {
+    const page = await newE2EPage({
+      html: `
+        <calcite-tree>
+          <calcite-tree-item id="cables">
+            Cables
+            <calcite-tree slot="children">
+              <calcite-tree-item id="xlr">XLR Cable</calcite-tree-item>
+              <calcite-tree-item id="instrument">Instrument Cable</calcite-tree-item>
+            </calcite-tree>
+          </calcite-tree-item>
+        </calcite-tree>
+    `
+    });
+
+    await page.keyboard.press("Tab");
+
+    expect(await page.evaluate(() => document.activeElement.id)).toBe("cables");
```

Comment:
Nitpick: you could return the values you're asserting on with the same `evaluate` callback.